ci/prow/make_config.go

```diff
@@ -109,13 +114,42 @@ type sectionGenerator func(string, string, yaml.MapSlice)
 // stringArrayFlag is the content of a multi-value flag.
 type stringArrayFlag []string
 
-var (
-	// Array constants used throughout the jobs.
-	allPresubmitTests = []string{"--all-tests", "--emit-metrics"}
-	releaseNightly    = []string{"--publish", "--tag-release"}
-	releaseLocal      = []string{"--nopublish", "--notag-release"}
+// ####################################################################################################
+// ############## data definitions that are used for the testgrid config file generation ##############
+// ####################################################################################################
+// baseTestgridTemplateData contains basic data about the testgrid config file.
+type baseTestgridTemplateData struct {
+	TestGroupName string
```

Comment:
You could use baseProwJobTemplateData instead of a separate structure, avoiding duplicate data and getting more info for generating the config. But it's not worth changing it now. Please add a TODO here about that for the future.